sample_xblocks/filethumbs/filethumbs.py

```diff
@@ -18,14 +19,18 @@
 
 log = logging.getLogger(__name__)
 
-arrow = ["11011",
-         "10001",
-         "00000",
-         "11011",
-         "11011",
-         "11011",
-         "10001"]
-arrow = [map(int, x) for x in arrow]
```

Comment:
Maybe, but I'm not looking to refactor the entire codebase at this point, so I'm considering it outside the scope of the submission. I'm only touching this to eliminate quality violations.